spec/getinline/transformer_spec.rb

```diff
@@ -45,5 +45,27 @@
         end
       end
     end
+
+    context 'with special characters' do
```

Comment:
We may want to refactor these down the line so we can test templates with both ERB tags and special characters - you asked about this before and I misunderstood the question.